src/themes/theme.ts

```diff
@@ -46,27 +47,31 @@ export default createTheme({
             dark: colors.purple[900],
         },
         info: {
-            light: colors.blue[700],
-            main: colors.blue[700],
-            dark: colors.blue[800],
+            light: colors.blue[50],
+            main: colors.blue[500],
+            dark: colors.blue[700],
         },
         success: {
-            light: colors.green[700],
-            main: colors.green[700],
+            light: colors.green[50],
+            main: colors.green[500],
             dark: colors.green[800],
         },
         warning: {
-            light: colors.orange[700],
-            main: colors.orange[700],
-            dark: colors.orange[800],
+            light: colors.orange[100],
+            main: colors.orange[800],
+            dark: colors.orange[900],
         },
         error: {
-            light: colors.red[700],
+            light: colors.red[50],
             main: colors.red[700],
             dark: colors.red[800],
         },
         divider: colors.grey[300],
         dividerAlternative: colors.grey[500],
+        tableHeaderHover: colors.grey[400],
+        highlight: '#FFEACC',
+        sidebarContainer: 'rgba(32,32,33, 0.2)',
+        sidebarDivider: 'rgba(255, 255, 255, 0.3)',
```

Comment:
Is this different than `theme.palette.divider` or `theme.palette.dividerAlternative`?